SW_Weather.h

```diff
@@ -58,16 +57,75 @@ typedef struct {
+    *PPT7thMon_mm,                            /**< Array containing July precipitation amount (mm) */
+
+    **meanTempMon_C,                        /**< 2D array containing monthly mean average daily air temperature (&deg;C)*/
+    **maxTempMon_C,                         /**< 2D array containing monthly mean max daily air temperature (&deg;C)*/
+    **minTempMon_C,                         /**< 2D array containing monthly mean min daily air temperature (&deg;C)*/
+    *meanTemp_C,                            /**< Array containing annual mean temperatures [C]*/
+    *meanTempDriestQtr_C,                   /**< Array containing the average temperature [C] of the driest quarter of the year*/
+    *minTemp2ndMon_C,                       /**< Array containing the mean daily minimum temperature in February [C] */
+    *minTemp7thMon_C,                       /**< Array containing minimum July temperatures [C] */
+
+    *frostFree_days,                        /**< Array containing the maximum consecutive days [-] without frost*/
+    *ddAbove65F_degday;                     /**< Array containing the amount of degree days [C x day] above 65 F */
+} SW_CLIMATE_YEARLY;
+
+/**
+ @brief A structure holding all variables that are output to the function `averageClimateAcrossYears()` #SW_CLIMATE_YEARLY
+ 
+ @note Values are across-year averages of #SW_CLIMATE_YEARLY and 1D array dimension represents month.
+ The exceptions are `sdC4` and `sdCheatgrass` which represent across-year standard devations and the 1D array dimension
+ represents different variables, see `averageClimateAcrossYears()`.
+ */
+typedef struct {
+    RealD *meanTempMon_C,                   /**< Array of size MAX_MONTHS containing sum of monthly mean temperatures*/
+    *maxTempMon_C,                          /**< Array of size MAX_MONTHS containing sum of monthly maximum temperatures*/
+    *minTempMon_C,                          /**< Array of size MAX_MONTHS containing sum of monthly minimum temperatures*/
+    *PPTMon_cm,                             /**< Array of size MAX_MONTHS containing sum of monthly mean precipitation*/
+    *sdC4,                                  /**< Array of size three holding the standard deviations of minimum July temperature (0),
+                                             frost free days (1), number of days above 65F (2)*/
+    *sdCheatgrass,                          /**< Array of size three holding the standard deviations of July precipitation (0), mean
+                                              temperature of dry quarter (1), mean minimum temperature of February (2)*/
+    meanTemp_C,                             /**< Value containing the average of yearly temperatures*/
+    PPT_cm,                                 /**< Value containing the average of yearly precipitation*/
+    PPT7thMon_mm,                            /**< Value containing average of July precipitation (mm)*/

```

Comment:
please update documentation throughout and replace "July" with 7th month (with explanation about north/south hemisphere)

Reply:
@N1ckP3rsl3y this still needs to be fixed, e.g.,
replace `Value containing average of July precipitation (mm)` (current) with something along the lines of `Value containing average precipitation (mm) in July (northern hemisphere) or in January (southern hemisphere)`,
and equivalently for the others